packages/components/TeamsList/TeamsList.js

```diff
@@ -54,7 +54,11 @@ const TeamsListComponent = ({ activeTeamId, handleToggle, setListsOpened }) => {
   return (
     <>
       {teamList.map(team => {
-        return activeTeamId === team?.id || !team?.id ? null : (
+        const isTeamHidden = activeTeamId === team?.id ||
+          !team?.id ||
+          !team.pinned;
```

Comment:
I think that here should be chain question `!team?.pinned;`

Reply:
Good catch